src/models/Admin.js

```diff
@@ -4,7 +4,8 @@ const AdminSchema = new mongoose.Schema({
   name: { type: String },
   // email should be unique and should be in  example@example.com formate match: /.+\@.+\..+/,
   email: { type: String, lowercase: true, required: true, match: /.+[\\@].+\..+/, unique: true },
-  password: { type: String }
+  password: { type: String },
+  avatar: { type: String }
 }, { timestamps: true })
 
 AdminSchema.index({ name: 'text', email: 'text' })
```

Comment:
Do we need a text index here? Don't we need two separate increasing indices for both name and email?